Pricing/ItemPriceCalculator.php

```diff
@@ -49,24 +95,17 @@ public function calculate($item, $currency = null, $useProductsPrice = true)
 
         // get bulk price
         if ($useProductsPrice) {
-            $product = $item->getCalcProduct();
-            $price = $this->priceManager->getBulkPriceForCurrency($product, $item->getCalcQuantity(), $currency);
-
-            // no price set - return 0
-            if ($price === null) {
-                return 0;
```

Comment:
yes, but it's not the same. the rest of the function would be skipped at this point with the return. 
but if it still works its fine